### ISSUE TITLE
hey iam tying to make neural network using keras with loss function of categorical_crossentropy  iam giving labels as(1080,6)but error occoured saying)

### ISSUE THREAD

**Issue by ahmedanis1** [keras-team/keras#12910]
The [keras.applications](https://github.com/keras-team/keras-applications) module has been splitted from the main Keras repository.

Please submit your issue using this [link](https://github.com/keras-team/keras-applications/issues/new).

Thank you!

**Comment by ahmedanis1** [comment]
the error is :
A target array with shape (1080, 6) was passed for an output of shape (None, 10) while using as loss `categorical_crossentropy`. This loss expects targets to have the same shape as the output.
model = tf.keras.models.Sequential([
  tf.keras.layers.Flatten(input_shape=(64, 64,3)),
  tf.keras.layers.Dense(512, activation=tf.nn.relu),
  tf.keras.layers.Dense(10, activation=tf.nn.softmax)
])
model.compile(optimizer='adam',
              loss='categorical_crossentropy',
              metrics=['accuracy'])
model.fit(train_x,train_y_2,epochs=10)

**Comment by jvishnuvardhan** [comment]
Please fill the issue template [here](https://github.com/keras-team/keras/issues/new/choose). Could you update them if they are relevant in your case, or leave them as N/A? Along with the template, please provide as many details as possible to find the root cause of the issue.  Thanks!

**Comment by jvishnuvardhan** [comment]
Closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
